torch_xla/__init__.py

```diff
@@ -182,3 +182,5 @@ def _init_xla_lazy_backend():
 if os.getenv('XLA_REGISTER_INSTALLED_PLUGINS') == '1':
   plugins.use_dynamic_plugins()
   plugins.register_installed_plugins()
+
+from .torch_xla import *
```

Comment:
why is this needed?

Reply:
This imports the contents of `torch_xla.py` into `torch_xla/`'s module scope. Otherwise, the functions would be `torch_xla.torch_xla.etc`. This assigns them to `torch_xla.etc`